doc/adr/0007-generate-qr-code-on-client

```diff
@@ -0,0 +1,25 @@
+# 1. Record architecture decisions
+
+Date: 2022-06-30
+
+## Status
+
+Accepted
+
+## Context
+
+QR Code is embedded in the eService page and it is used to allow user to start identification process via the UseID app camera which later. The QR Code contains `tcTokenURL`, `widgetSessionId` and `encryptionKey`. For this decision record, [ADR 0006](LINK TO 0006) 
```

Comment:
There are probably a few words too much? "camera which later"
But it should better be "camera app" instead of "UseID app".
And the link intro should be: "For this decision, see record ..."

So all in all maybe:
```suggestion
A QR code is embedded in the eService page via iframe widget and it is used to allow user to start identification process via camera app on a mobile device. The QR Code contains `tcTokenURL`, `widgetSessionId`, a `mobile` flag and an `encryptionKey`. (For details see [ADR 0006](LINK TO 0006).)
```